### ISSUE TITLE
Errors when running the application with no active games/during the offseason

### ISSUE THREAD

**Issue by cja769** [paaatrick/playball#8]
I can't confirm if this happens on days when there's no baseball games during the season (regular or post) however right now during the offseason the following happens

When the list of games (or lack thereof) are suppose to be shown, this error is thrown
```
The above error occurred in the <GameList> component:
    in GameList (created by Connect(GameList))
    in Connect(GameList) (created by App)
    in element (created by App)
    in element (created by App)
    in App (created by Connect(App))
    in Connect(App)
    in Provider

React will try to recreate this component tree from scratch using the error boundary you provided, App.
Warning: App: Error boundaries should implement getDerivedStateFromError(). In that method, return a state update to display an error message or fallback UI.
Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method.
    in LoadingSpinner (created by Connect(LoadingSpinner))
    in Connect(LoadingSpinner) (created by HelpBar)
    in element (created by HelpBar)
    in HelpBar (created by App)
    in element (created by App)
    in element (created by App)
```
then if you type `L` then the application exist and shows this error
```
TypeError: Cannot read property 'games' of undefined
    at GameList.handleGameSelect (/home/jay/.nvm/versions/node/v10.16.0/lib/node_modules/playball/dist/components/GameList.js:111:40)
    at eventListener (/home/jay/.nvm/versions/node/v10.16.0/lib/node_modules/playball/node_modules/react-blessed/dist/index.js:106:5)
    at List.instance._eventListener.args [as _eventListener] (/home/jay/.nvm/versions/node/v10.16.0/lib/node_modules/playball/node_modules/react-blessed/dist/index.js:2808:46)
    at List.EventEmitter._emit (/home/jay/.nvm/versions/node/v10.16.0/lib/node_modules/playball/node_modules/blessed/lib/events.js:94:20)
    at List.EventEmitter.emit (/home/jay/.nvm/versions/node/v10.16.0/lib/node_modules/playball/node_modules/blessed/lib/events.js:111:8)
    at List.enterSelected (/home/jay/.nvm/versions/node/v10.16.0/lib/node_modules/playball/node_modules/blessed/lib/widgets/list.js:586:8)
    at List.<anonymous> (/home/jay/.nvm/versions/node/v10.16.0/lib/node_modules/playball/node_modules/blessed/lib/widgets/list.js:119:14)
    at List.EventEmitter._emit (/home/jay/.nvm/versions/node/v10.16.0/lib/node_modules/playball/node_modules/blessed/lib/events.js:94:20)
    at List.EventEmitter.emit (/home/jay/.nvm/versions/node/v10.16.0/lib/node_modules/playball/node_modules/blessed/lib/events.js:117:12)
    at Program.<anonymous> (/home/jay/.nvm/versions/node/v10.16.0/lib/node_modules/playball/node_modules/blessed/lib/widgets/screen.js:602:15)
```

This is reproducible by just running the `playball` command right now as it's the offseason.

Of course this is a very small thing but something I observed.

**Comment by salmaniqbal** [comment]
First of all @paaatrick, this is amazing! I don't even watch MLB but now I will 😄 ⚾️ 

I think @cja769 is right, I got the same error when there are no games on.

**Comment by toreau** [comment]
I can confirm. Would be nice to see upcoming/finished games when nothing else is going on.

**Comment by kangaree** [comment]
Confirming here too. Would be nice if you could return to the playoffs- World Series 2019 or something.

I came across this from HN and it's a shame it showed up when the season was over.

**Comment by paaatrick** [comment]
Thanks! Should be fixed in version 2.0.4.

**Comment by cja769** [comment]
Thanks @paaatrick. It's a cool application. Can't wait to use it next season
